.circleci/config.yml

```diff
@@ -47,16 +47,196 @@ jobs:
+            echo "deb https://packages.cloudfoundry.org/debian stable main" | tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+            apt-get update
+            apt-get install cf-cli
+      - run:
+          name: Deploy to dev
+          command: |
+              ./blue-green-deploy.sh $USERNAME_DEV $CF_PASSWORD_DEV dev
+
+  deploy-stage:
+    working_directory: ~/code
+    docker:
+      # CircleCI Python images available at: https://hub.docker.com/r/circleci/python/
+      - image: python:3.7
+        environment: # environment variables for primary container
+          PIPENV_VENV_IN_PROJECT: true
+          DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
+      # CircleCI PostgreSQL images available at: https://hub.docker.com/r/circleci/postgres/
+      - image: postgres:10.1-alpine
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: circle_test
+    steps:
+      - checkout
+      - restore_cache:
+          # this should all be cached from the test
+          key: deps9-{{ .Branch }}-{{ checksum "Pipfile.lock" }}-{{ checksum "package.json" }}
+      - run:
+          name: Node install
+          command: |
+            echo "deb https://deb.nodesource.com/node_10.x stretch main" > /etc/apt/sources.list.d/nodesource.list && \
+            wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
+            echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list && \
+            wget -qO- https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+            apt-get update && \
+            apt-get install -yqq nodejs yarn && \
+            npm i -g npm@^6 && \
+            rm -rf /var/lib/apt/lists/*
+            npm install
+      - run:
+          # database will be migrated in cloud.gov
+          name: Python install with pipenv
+          command: |
+            pip install -U pip && pip install pipenv && \
+            pip install pipenv
+            pipenv install
+      - run:
+          name: Install cloud foundry deploy tools
+          command: |
+            wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | apt-key add -
+            echo "deb https://packages.cloudfoundry.org/debian stable main" | tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+            apt-get update
+            apt-get install cf-cli
+      - run:
+          name: Deploy to stage
+          command: |
+            ./blue-green-deploy.sh $USERNAME_STAGE $CF_PASSWORD_STAGE stage
+
+  deploy-prod:
+    working_directory: ~/code
+    docker:
+      # CircleCI Python images available at: https://hub.docker.com/r/circleci/python/
+      - image: python:3.7
+        environment: # environment variables for primary container
+          PIPENV_VENV_IN_PROJECT: true
+          DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
```

Comment:
@LindsayYoung Do we want to change this for `deploy-prod`?

Reply:
It should be fine because we aren't deploying this image, we are using the image to compile the code, the real database url etc is set in cloud.gov.

Reply:
@LindsayYoung Makes sense that this will be overwritten, since the real URL is set in cloud.gov! Sounds like we can probably remove this line safely in that case, but let's do that in a follow-on PR. 🚢